### PR TITLE
cvmfs_cache_posix: packaging, test & minor bugfix

### PR DESCRIPTION
**Pull request by janpriessnitz**

### CODE REVIEW

File: test/src/089-external_cache_plugin/main [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  fi
+}
+
+cvmfs_run_test() {
+  logfile=$1
+
+  trap cleanup EXIT HUP INT TERM
+
+  mount_opts="CVMFS_CACHE_PRIMARY=externalplugin"
+  mount_opts="$mount_opts CVMFS_CACHE_externalplugin_TYPE=external"
```
Comment (jblomer) [inline test/src/089-external_cache_plugin/main]: Isn't there a linebreak missing?

**Comment by jblomer** [comment]
The integration tests should be excluded from the ram cache "platform"

File: test/src/089-external_cache_plugin/main (hunk 1 above)
Comment (janpriessnitz) [reply test/src/089-external_cache_plugin/main]: it's delimited by space. These opts are later passed to cvmfs_mount()

Comment (jblomer) [reply test/src/089-external_cache_plugin/main]: Ah, thanks

**Pull request merged**